.vscode/tasks.json

```diff
@@ -73,22 +73,13 @@
         "command": "dotnet",
         "args": [
           "pwsh",
+          "-noprofile",
           "${workspaceFolder}/scripts/vscode-build.ps1",
           "-filePath",
           "${file}",
           "-msbuildEngine",
           "dotnet"
         ],
-        "windows": {
-          "command": "powershell",
```

Comment:
There is no need for this anymore as part of our repo setup includes installing `pwsh` as a local tool.